armi/operators/tests/test_operators.py

```diff
@@ -543,32 +543,44 @@ def test_getMaxBurnSteps(self):
 
 
 class TestInterfaceAndEventHeaders(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.o, cls.r = test_reactors.loadTestReactor(
+            inputFileName="smallestTestReactor/armiRunSmallest.yaml",
```

Comment:
`armiRunSmallest.yaml`!

You're the first person (other than me) to use it! Nice!

Reply:
well, it was easy to use....since you had already edited this file to use it! copy-paste for the win!

![image](https://github.com/terrapower/armi/assets/5347433/d9511512-3910-4c61-9c7e-c9672906f1e1)